### PR TITLE
ENH: added sortmerna

### PR DESCRIPTION
**Pull request by wasade**
It's possible that some testing of presence of darwin vs. linux will be necessary, but here's at least a first pass.

### CODE REVIEW

**Comment by wasade** [comment]
Just to tag it, resolves #20

**Comment by RNAer** [comment]
@wasade , would you mind rebasing it on top of the repo? We fixed a series of bugs in last commits.

File: recipes/SortMeRNA/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  url: ftp://ftp.microbio.me/pub/sortmerna-2.0-no-db.tar.gz
+ 
+requirements:
+  build:
+    - gcc
+ 
+test:
+  commands:
+    - sortmerna -h
+
```
Comment (RNAer) [inline recipes/SortMeRNA/meta.yaml:12]: This `requirements` section is usually not necessary for `gcc`.

**Comment by RNAer** [comment]
@wasade , `Error: package/name must be lowercase, got: u'SortMeRNA'`

**Comment by RNAer** [comment]
it is close. the new error is because the return code of `sortmerna -h` is 1 instead of 0. conda uses that code to judge if the test passes.

**Comment by wasade** [comment]
Thanks. Created biocore/sortmerna#96 as I don't think that is correct behavior.

**Comment by Jorge-C** [comment]
Great, thanks!

**Pull request merged**